user-defined-plugin.md

```diff
@@ -238,7 +226,7 @@ If you want to make a parameter as mandatory parameter, set the `required` attri
 
 ## Create plugin R script file.
 
-If your plugin simply passes input parameter values from UI to the R function defined in `function` attribute, you can stub the R script. (i.e. all you need is create a single line `library(riem)` file and save it as `plugin.r`)
+If your plugin simply passes input parameter values from UI to the R function defined in `function` attribute, you don't need to define a new function in the R script. (i.e. all you need is create a single line `library(riem)` file and save it as `plugin.r`)
```

Comment:
"to the R function defined in `function` attribute"
To be clear that this is for the case about already existing function, I think something like following would be clearer.
"to an existing R function pointed by `function` attribute"